src/icons/icons/svgs/ArrowIcon.tsx

```diff
@@ -3,6 +3,8 @@ import * as React from 'react';
 function SvgArrowIcon(props: React.SVGProps<SVGSVGElement>) {
   return (
     <svg
+      width={16}
+      height={16}
       viewBox="0 0 46.29 32.92"
```

Comment:
I guess this can be the only exception, I still think we should move this out of our Icons library. I'll take a look after v16 is released

Reply:
I left a comment in Figma about this for Ben to resolve last week. If you'd like to bring it up with him again you're welcome to but I consider the matter low-priority.

Reply:
Yeah saw that, I actually was always thinking on removing from public Radiance. I added this icon and Ida created it for me after trying JPG and PNGs. I don't think design intention was to put it here ever, it was my bad to be able to use the SVGR generation. Ill take a look in Cooldown,

Reply:
I know there is directly usage in Curology Ill figure something out